scripts/zones/Arrapago_Reef/mobs/Medusa.lua

```diff
@@ -1,28 +1,45 @@
 -----------------------------------
 -- Area: Arrapago Reef
 -- NPC:  Medusa
+-- TODO: resists, attack/def boosts
 -----------------------------------
 
 require("scripts/globals/titles");
+require("scripts/zones/Arrapago_Reef/MobIDs");
+require("scripts/zones/Arrapago_Reef/TextIDs");
 
 -----------------------------------
 -- onMobSpawn Action
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:setLocalVar("eeshpp", math.random(5,99)); -- Uses EES randomly during the fight
 end;
 
 -----------------------------------
 -- onMobEngaged Action
 -----------------------------------
 
-function onMobEngaged(mob,target)
+function onMobEngaged(mob, target)
+    target:showText(mob, MEDUSA_ENGAGE);
+    SpawnMob(ExonOne,   180):updateEnmity(target);
+    SpawnMob(ExonTwo,   180):updateEnmity(target);
+    SpawnMob(ExonThree, 180):updateEnmity(target);
+    SpawnMob(ExonFour,  180):updateEnmity(target);
+end;
 
-	SpawnMob(16998863,180):updateEnmity(target);
-	SpawnMob(16998864,180):updateEnmity(target);
-	SpawnMob(16998865,180):updateEnmity(target);
-	SpawnMob(16998866,180):updateEnmity(target);
+-----------------------------------
+-- onMobFight Action
+-----------------------------------
 
+function onMobFight(mob, target)
+    local HPP = mob:getHPP();
+    if (mob:getLocalVar("usedees") == 0) then
+        if (HPP <= mob:getLocalVar("eeshpp")) then
+            mob:useMobAbility(479); -- Eagle Eye Shot

```

Comment:
I'm not saying mob skill id's could change from the client side. I'm saying we don't yet have all the two hour mob skills, for example there is an EES animation for each family, so we'll have to add a new mob skill for each. I just don't like the idea of hard coding the skill id when we don't have to.

We can push the two hour logic to scripts for sure, but we need better control for two hours, something to of course remember if it was already used (no need to manage yourself). We'll need something like: `mob:useTwoHour(JOB_THF)`, (Use two hour regardless if already used) `mob:hasUsedTwoHour(JOB_THF)`, `mob:resetTwoHour(JOB_THF)`.  (with no args it could default to current main job). 

We don't need new bindings for this (but I would still be for adding them to the mob, it's just cleaner). Could add a new `mob` or `mobutils` namespace for these types of methods. 

```
if(mobutils.hasUsedTwoHour(mob, JOB_THF) == false) then
mobutils.useTwoHour(mob, JOB_THF)
end
``` etc..